stockflux-securities-master/src/components/confirmation/Confirmation.js

```diff
@@ -1,10 +1,13 @@
 import React, { useState } from "react";
 import "./Confirmation.css";
-import Button, { ButtonSize } from "./Button";
+import Button, { ButtonSize } from "../button/Button";
 import PropTypes from "prop-types";
 
 const Confirmation = ({ children, confirmationText }) => {
   const [clickedStatus, setClickedStatus] = useState(false);
+  const handleClickNoButton = () => {
+    setClickedStatus(false);
+  };
 
   return (
     <div className="confirmation-button-container">
```

Comment:
Hmm, I tend to avoid naming classes containers, unless that class wraps some component. What I mean is that usually a class with a word "container" in it is used in the parent component to, e.g. position the child element. Aka I think it should not be inside the component itself, but again not sure. @JoelMagee what do you think?

Reply:
Was this mention meant for @joelwilliamsmith ;)

Reply:
Yes, indeed! Sorry for the mixup, not sure how this happened!

Reply:
It probably shouldn't be named "confirmation-button-container" either because it isn't a button component anymore. Maybe just "confirmation". 

 In the same component I have "confirmation-message-container" and "confirmation-option-button-container" as well so I'll drop the word container on each of these?

Reply:
I've made this name change now but will wait for Joel to comment.

Reply:
I've now made the change to include shortening the classNames in many of the other components in which this kind of thing is currently done.